.github/find-hotfix-tags.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+tag_regex='v[0-9]+\.[0-9]+\.[0-9]+-osghotfix-[A-Za-z]+'
+git tag -l | sort | egrep $tag_regex > git_tags
+
+token=$(curl -s \
+             -H "Content-Type: application/json" \
+             -X POST \
+             -d '{"username": "'${DOCKER_USERNAME}'", "password": "'${DOCKER_PASSWORD}'"}' \
+             https://hub.docker.com/v2/users/login/ | jq -r .token)
+
+curl -s \
+     -H "Authorization: JWT ${token}" \
+     https://hub.docker.com/v2/repositories/brianhlin/stash-cache/tags/?page_size=100 | \
```

Comment:
The `?` is not likely to cause you any trouble here, but technically it is a glob wildcard character, so it's kinda better to quote this url in the command line.

(It becomes a bigger deal for url's with `&`'s, as those do get interpreted to break up the command and put everything to the left in the background if they're unquoted.)